### DISCUSSION TITLE
Unable to figure out how to add a FOSSHUB source

### DISCUSSION THREAD

**Discussion by xcpn** [ImranR98/Obtainium#1613] [Q&A]
Is it possible to add a [FOSSHUB](https://www.fosshub.com) source, because I can't figure it out and it seems like there are no direct links for the assets.

I'm trying to add [qBittorrent](https://www.fosshub.com/qBittorrent.html) (specifically the AppImage) and it doesn't work.

Any help appreciated. Also I would like to suggest FOSSHUB as an official source through the app, if it's possible.

**Comment by xcpn** [comment]
@ImranR98 is it possible to add FOSSHUB as an official source in the app, or is it even possible to add it manually?

I tried to but to no avail. It has link protection to prevent automation if I'm looking it correctly.

**Comment by ImranR98** [comment]
Fosshub uses javascript to generate unique links per download, so obtainium can't scrape it.

**Comment by xcpn** [comment]
@ImranR98 unfortunate!
